src/Components/Breakdown.svelte

```diff
@@ -132,6 +153,29 @@
 				</div>
 			{/each}
 		</details>
+		<details class="non-existings">
```

Comment:
feel free to change styling as desired. It can be done in this PR or a future one. For now I just borrowed the same styles as everything else.